solidity/contracts/deposit/DepositUtils.sol

```diff
@@ -489,8 +489,10 @@ library DepositUtils {
         bool frtExists = feeRebateTokenHolder(_d) != address(0);
 
         uint256 frtAdjustment = computeFrtAdjustment(_d, redeemerOwesNoFee, redeemerHoldsFrt, frtExists);
```

Comment:
Sadly this means redeemerOwesNoFee is no longer the correct term here. Maybe thirdPartyRedeemerOwesNoFee?

Reply:
ah yes

Reply:
how about `redeemerOwesNoFeeRebate`?